packages/eslint-config-base/src/rules/best-practices.ts

```diff
@@ -0,0 +1,452 @@
+    'no-unused-expressions': [
+      'error',
+      {
+        allowShortCircuit: false,
+        allowTernary: false,
+        allowTaggedTemplates: false,
+      },
+    ],
+
+    // disallow unused labels
+    // https://eslint.org/docs/rules/no-unused-labels
+    'no-unused-labels': 'error',
+
+    // disallow unnecessary .call() and .apply()
+    // https://eslint.org/docs/rules/no-useless-call
+    'no-useless-call': 'off',
+
+    // Disallow unnecessary catch clauses
+    // https://eslint.org/docs/rules/no-useless-catch
+    'no-useless-catch': 'error',
+
+    // disallow useless string concatenation
+    // https://eslint.org/docs/rules/no-useless-concat
+    'no-useless-concat': 'error',
+
+    // disallow unnecessary string escaping
+    // https://eslint.org/docs/rules/no-useless-escape
+    'no-useless-escape': 'error',
+
+    // disallow redundant return; keywords
+    // https://eslint.org/docs/rules/no-useless-return
+    'no-useless-return': 'error',
+
+    // disallow use of void operator
+    // https://eslint.org/docs/rules/no-void
+    'no-void': 'error',
+
+    // disallow usage of configurable warning terms in comments: e.g. todo
+    // https://eslint.org/docs/rules/no-warning-comments
+    'no-warning-comments': [
+      'off',
+      { terms: ['todo', 'fixme', 'xxx'], location: 'start' },
+    ],
+
+    // disallow use of the with statement
+    // https://eslint.org/docs/rules/no-with
+    'no-with': 'error',
+
+    // require using Error objects as Promise rejection reasons
+    // https://eslint.org/docs/rules/prefer-promise-reject-errors
+    'prefer-promise-reject-errors': ['error', { allowEmptyReject: true }],
+
+    // Suggest using named capture group in regular expression
+    // https://eslint.org/docs/rules/prefer-named-capture-group
+    'prefer-named-capture-group': 'off',
+
+    // https://eslint.org/docs/rules/prefer-regex-literals
+    'prefer-regex-literals': [
+      'error',
+      {
+        disallowRedundantWrapping: true,
+      },
+    ],
+
+    // require use of the second argument for parseInt()
+    // https://eslint.org/docs/rules/radix
+    radix: 'error',
+
+    // require `await` in `async function` (note: this is a horrible rule that should never be used)
+    // https://eslint.org/docs/rules/require-await
+    'require-await': 'off',
+
+    // Enforce the use of u flag on RegExp
+    // https://eslint.org/docs/rules/require-unicode-regexp
+    'require-unicode-regexp': 'off',
+
+    // requires to declare all vars on top of their containing scope
+    // https://eslint.org/docs/rules/vars-on-top
+    'vars-on-top': 'error',
+
+    // require immediate function invocation to be wrapped in parentheses
+    // https://eslint.org/docs/rules/wrap-iife.html
+    'wrap-iife': ['error', 'outside', { functionPrototypeMethods: false }],
+
+    // require or disallow Yoda conditions
+    // https://eslint.org/docs/rules/yoda
+    yoda: 'error',
+  },
+};
+
+export = config;
```

Comment:
Why are we using `export =` here?

Reply:
This repo is in CJS, we cannot use `export default` here